test/clj/rems/test_browser.clj

```diff
@@ -226,7 +226,16 @@
   ([]
    (get-attachments {:css "a.attachment-link"}))
   ([selector]
-   (mapv (partial btu/get-element-text-el) (btu/query-all selector))))
+   (mapv btu/get-element-text-el (btu/query-all selector))))
+
+(defn get-validation-summary []
+  (mapv btu/get-element-text-el (btu/query-all {:css "#flash-message-top ul li"})))
+
+(defn get-validation-for-field [label]
+  (let [el (first (btu/query-all [{:css ".fields"}
```

Comment:
I think `query` does what this `first ... query-all` does too but since it's not very specific in its description about it maybe it's best to be explicit like you have put here and we have elsewhere.

Reply:
Good point, the query-all was a copy-paste leftover from `fill-form-field`